app/src/main/java/com/example/bbltripplanner/screens/posting/composables/PostingInitScreen.kt

```
@@ -1,22 +1,473 @@
+        }
+    }
+}
+
+enum class BottomSheetType {
+    USER_SELECTION,
+    LOCATION_SELECTION
+}
+
+fun moveToNextPage(navController: NavController) {
+
+}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)